manager_dashboard/manager_dashboard/js/usergroup-management.js

```diff
@@ -0,0 +1,115 @@
+$("#add-usergroup-modal").on('show.bs.modal', function() {
+  $('#usergroup-name').val('');
+  $('#usergroup-description').val('');
+})
+
+function submitNewUserGroup() {
+  var nameInput = $('#usergroup-name');
+  var name = (nameInput.val() || '').trim();
+  var descriptionInput = $('#usergroup-description');
+  var description = (descriptionInput.val() || '').trim();
+
+  if (name && name.length <= 50) {
+    $('#add-usergroup-modal-loading-message-container').removeClass('hidden');
+    nameInput.attr('disabled', true);
+    descriptionInput.attr('disabled', true);
+
+    firebase.database().ref('v2/userGroups/' + name).once("value", function(snapshot) {
```

Comment:
Please do some validation on the value input by user before using it like this.

Reply:
Done!